src/platform/plugins/shared/console/server/lib/spec_definitions/js/retriever.ts

```diff
@@ -105,6 +105,28 @@ export const retriever = (specService: SpecDefinitionsService) => {
       match_criteria: {},
       rank_window_size: 10,
     },
+    pinned: {
+      __template: {
+        retriever: {},
+        ids: [],
+        docs: [],
```

Comment:
for the template - we should only keep one, either ids or docs - let's keep `ids`?
the reason for this is because we can only use `ids` or `docs` but not both, because the API would return an error